apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/wpcom-global-styles-view.js

```diff
@@ -9,11 +9,22 @@ function recordEvent( button ) {
 	} );
 }
 
-( function () {
-	const popoverToggle = document.querySelector( '.launch-bar-global-styles-toggle' );
-	const popover = document.querySelector( '.launch-bar-global-styles-popover' );
-	const upgradeButton = document.querySelector( '.launch-bar-global-styles-upgrade-button' );
-	const previewButton = document.querySelector( '.launch-bar-global-styles-preview-link' );
+document.addEventListener( 'DOMContentLoaded', () => {
```

Comment:
Really minor drive-by comment but there's a [`wp.dom-ready`](https://github.com/WordPress/gutenberg/tree/3245d1e0518d6a2fa565266981fe0fab8a984a8e/packages/dom-ready#readme) helper which is a bit more thorough than standard `DOMContentLoaded` listener.